streaming/python/ray/streaming/processor.py

```diff
@@ -0,0 +1,224 @@
+from __future__ import absolute_import
```

Comment:
Consider splitting this out into one file per operator and putting it in some operators directory.

Reply:
We'll have a new operators implementation in later pr. Maybe we can put it later.